rfc-merge-procedure.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: default
+title: RFC Merge Procedure
+---
+
+Once an RFC has been accepted (i.e., the final comment period is
```

Comment:
Probably good to clarify that this is after an FCP *where no serious new issues/arguments were raised*.